libs/platform/i18n/src/lib/locale/adapter/default-locale.adapter.ts

```diff
@@ -0,0 +1,39 @@
+import { inject } from '@spryker-oryx/di';
+import { Observable, of } from 'rxjs';
+import { Locale } from '../../models';
+import { LocaleAdapter } from './locale.adapter';
+
+export class DefaultLocaleAdapter implements LocaleAdapter {
+  constructor(
+    protected readonly config = inject(DefaultLocaleAdapterConfig, null)
+  ) {}
+
+  getAll(): Observable<Locale[]> {
+    return of(this.config?.locales ?? []);
+  }
+
+  getDefault(): Observable<string> {
+    return of(
+      this.config?.defaultLocale ??
+        this.config?.locales?.[0].code ??
+        this.getFallbackLang()
+    );
+  }
+
+  protected getFallbackLang(): string {
+    return navigator.language;
+  }
+}
+
+export interface DefaultLocaleAdapterConfig {
+  readonly locales?: Locale[];
+  readonly defaultLocale?: string;
+}
+
+export const DefaultLocaleAdapterConfig = 'oryx.DefaultLocaleAdapterConfig';
```

Comment:
Should it not be `LocaleAdapterConfig`?

Reply:
We discussed with Stan and will bring this topic to catchup as there are some things to consider.
For now we will leave it as is.